tests/js/common/shell/shell-fs.js

```diff
@@ -73,7 +73,7 @@ function FileSystemSuite () {
 ////////////////////////////////////////////////////////////////////////////////
 
     tearDown : function () {
-      // some sanity check as we don't want to unintentionally remove "." or "/"
+      // some basic check as we don't want to unintentionally remove "." or "/"
```

Comment:
integrity

Reply:
This is rather a "security" check, so I will adjust it.